src/org/nyanya/android/traditionalt9/CharMap.java

```diff
@@ -108,35 +116,44 @@
 
 	protected static final char[][] FRT9TABLE = {
 		{ '0', '+' },
-		{ '.', ',', '?', '!', ':', ';', '"', '/', '-', '@', '^', '€', '$', '%', '&', '*', '#',  '(', ')', '_', '1' },
-		{ 'a', 'b', 'c', 'A', 'B', 'C', 'ä', 'Ä','á', 'â', 'à', 'å', 'ç', 'Á', 'Â', 'À', 'Å', 'Ç', '2' },
```

Comment:
Are these characters not needed to French input?

Reply:
ä, á, å do not exist in French input; æ exists in French input.
See https://fr.wikipedia.org/wiki/Alphabet_fran%C3%A7ais

Reply:
Okay understood. Thanks for looking in to that.